.github/workflows/rust.yml

```diff
@@ -82,6 +82,36 @@ jobs:
     - name: Run Clippy Lints
       run: cargo xtask clippy
 
+  check-features:
```

Comment:
Do you plan to make this feature required for merging to `main`?

Reply:
I would say yes, but that's usually done via settings, right (which I can't access on the repo?)?

Reply:
Ok, I think that makes sense. I'll make you an admin, and you can twiddle the settings after merge to main.